### DISCUSSION TITLE
How to set the user/group of the frankenphp server process

### DISCUSSION THREAD

**Discussion by andersonpem** [php/frankenphp#579] [Q&A] [answered]
I'm creating a dev container that mirrors the permissions of the user in the computer, so the user can work without having permission denied errors in the webserver.

My question is: can I set a user/group for frankenphp to use?

**Comment by dunglas** [comment, marked as answer]
Yes! https://frankenphp.dev/docs/docker/#running-as-a-non-root-user

**Comment by andersonpem** [comment]
Oh, shoot! I looked at the docs and didn't find this one. Thank youuuuuu!
